Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -31,24 +64,44 @@ public void OnGUI()
 		if (this.enabled && !this.isMinimized) {
 
 			float frameWidth = Screen.width / 3;
-			Rect frame = new Rect (10, 10, frameWidth, Screen.height);
+			Rect frame = new Rect (10, 10, frameWidth, Screen.height / 2);
 
 			GUI.BeginGroup (frame);
 			GUIStyle style = new GUIStyle (GUI.skin.box);
 			style.wordWrap = true;
 			style.fixedWidth = frameWidth;
 			GUILayout.Box (this.content, style);
+
+			if (isDecisionNode) {
+				GUIStyle decisionHintStyle = new GUIStyle (style);
+				decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+				if (!isOptionsGuiOpen) {
+					GUILayout.Box ("Press Q to progress in the story...", decisionHintStyle);
+				} else {
+					GUILayout.Box ("Press Q to scroll, E to close, ENTER to choose", decisionHintStyle);
```

Comment:
I'm not crazy about this input style - but I have no alternative idea to suggest. All our best keyboard keys are already bound to things. 

:(